src/js/components/form-elements/v2/ButtonFileSelect.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+
+import notification from 'components/Layout/notifications/notification';
+import NotificationType from 'consts/notificationTypes';
+import useTranslate from 'hooks/useTranslate';
+import Translate from 'utils/Translate';
+
+const ButtonFileSelect = ({
```

Comment:
Since we have different components for "Button" and "Dropzone", then perhaps the other one should be renamed to `DropzoneFileSelect` too.